Makefile

```diff
@@ -65,3 +65,8 @@ help: ## Show this message
 		sed 's/:[^#]*[#]# /|/'		| \
 		sed 's/%/LANG/'			| \
 		column -t -s'|' >&2
+
+.PHONY: test
+test: ## Run the tests
+	make upm
```

Comment:
```suggestion
```
Do we need to build upm to run the tests?

Reply:
We need the `statik` folder to run the tests, otherwise it won't compile. I'm not the biggest make expert, would copying the `go run github.com/rakyll/statik -src resources -dest internal -f` line from `make upm` do the same thing?

Reply:
Ah I see, we could probably split that out to its own make target and have both `cmd/upm/upm` and `test` depend on it.

For now we could also do:
```go
.PHONY: test
test: cmd/upm/upm ## Run the tests
    go test ./... -v
```
This will do the exact same thing as what you have but in a slightly more idiomatic way

Reply:
Having the statik command as a separate target and making it a dep of both build and test seems to work :smile: